doc/reference/peripherals/i2c.rst

```diff
@@ -1,12 +1,58 @@
 .. _i2c_api:
 
-
 I2C
 ####
 
 Overview
 ********
 
+.. note::
+
+   Zephyr recognizes the need to change the racially-charged terms
```

Comment:
IMO this type of note should be done in one place in the Zephyr documentation and not per peripheral.